src/main/java/net/cryptic_game/microservice/network/model/Invitation.java

```diff
@@ -30,7 +30,7 @@
     private UUID network;
     private boolean request;
 
-    private Invitation(UUID uuid, UUID device, UUID user, UUID network, boolean request) {
+    public Invitation(UUID uuid, UUID device, UUID user, UUID network, boolean request) {
```

Comment:
This is bad!
Do not change the access visibility for testing.

Reply:
This won't be changed